src/CIQS-CLI-Extension/azext_ciqs/format.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from collections import OrderedDict
+from collections import defaultdict
+
+def transform_templateListItem(result):
```

Comment:
This is a private method, isn't it? Suggest to:
- make it private;
- rename "transform_templateListItem(result)" to "transform_templateListItem(template_list)"; 
- rename "transform_templateList(template_list)" to "transform_templateList(result)";